app/src/main/java/edu/uc/app/parkfinder/service/ParkService.kt

```diff
@@ -10,8 +11,8 @@ import retrofit2.Response
 
 class ParkService {
 
-    fun fetchParks() : MutableLiveData<ArrayList<Park>> {
-        var _parks = MutableLiveData<ArrayList<Park>>()
+    internal fun fetchParks() : MutableLiveData<ArrayList<Park>> {
```

Comment:
Changed visibility to internal https://kotlinlang.org/docs/visibility-modifiers.html